### PR TITLE
Add new ImageTest (FINERACT-1218)

### PR DESCRIPTION
**Pull request by vorburger**
see FINERACT-1218

### CODE REVIEW

**Comment by vorburger** [comment]
@vidakovic @ptuomola this one is next.. do you both want to review and LGTM this, and the 2nd one merges when good?

PS: This will conclude & resolve FINERACT-1218!

**Comment by vidakovic** [comment]
@vorburger there seems to be an import error?

File: fineract-client/src/main/java/org/apache/fineract/client/util/FineractClient.java
Comment (vidakovic) [inline fineract-client/src/main/java/org/apache/fineract/client/util/FineractClient.java]: @vorburger  This import seems to fail

**Comment by vorburger** [comment]
> @vorburger there seems to be an import error?

Fixed that, but I expect that this will now fail due to FINERACT-1242... hereby calling out looking for anyone willing to fix that!

**Comment by vorburger** [comment]
@vidakovic @ptuomola review this one? Finally green build.

**Review by vidakovic — APPROVED** [review]
LGTM

**Review by ptuomola — APPROVED** [review]
LGTM

**Pull request merged**
